label_studio/projects/models.py

```diff
@@ -857,7 +857,9 @@ def _update_tasks_counters_and_is_labeled(self, task_ids, from_scratch=True):
         num_tasks_updated = 0
         page_idx = 0
 
-        while (task_ids_slice := task_ids[page_idx * settings.BATCH_SIZE:(page_idx + 1) * settings.BATCH_SIZE]):
+        batch_size = settings.BATCH_SIZE or len(task_ids)
```

Comment:
Really I'm not sure we need it. I checked this setting and it's static, users can't override it: https://github.com/heartexlabs/label-studio/blob/develop/label_studio/core/settings/base.py#L483

Reply:
Ah, sounds good, I'm convinced. I didn't know if there was some other override mechanism in place for these settings.